standard.tex

```diff
@@ -98,54 +107,42 @@ \subsection{PCB Design}
 \end{figure}
 
 \item Don't print PCBs at the 3D Printing Center in E5. These are typically poor quality.
+\item Make sure that soldermask clearance is set such that there is soldermask between component pads (pay attention to fine pitched components) and that there is no mask on the pads themselves.
+\item Thoroughly check the Gerber files for the board before you send them out for fabrication.
```

Comment:
Why? I mean, I know why, but we should elaborate that the gerbers are the be all end all final product - reviews happen on the gerbers with a commit hash associated with them and they are the same regardless of machine.

Also, what do I check them for?

Reply:
What are the exact changes you want made?

Reply:
fixed